.github/integration/tests/rabbitmq/20_configuration_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -ex
+
+MQHOST="localhost:15672"
+if [ -f /.dockerenv ]; then
+	MQHOST="mq:15671"
+fi
+
+timeout=$(curl -s --cacert /tmp/certs/ca.crt -u guest:guest "https://$MQHOST/api/queues/sda/from_cega" | jq '."consumer_details"[]."consumer_timeout"')
+if [ "$timeout" -ne 1000 ]; then
+    echo "active timeout is wrong, expected: 1000, actual: $timeout"
+    exit 1
```

Comment:
Shouldn't this number be `14400000` instead of `1000`? What am I missing?

Reply:
If you  look in `rabbitmq-federation.yml` you will see that for the tests we set the timeout to 1s , just to test that the modified value actually gets enabled.

Reply:
Oh, I missed line102 there!